travis/travis_after_success_pypi

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+pip install twine
+
+echo "[pypi]"  > $HOME/.pypirc
+echo "repository: https://pypi.python.org/pypi" >> $HOME/.pypirc
+echo "username: laslabs" >> $HOME/.pypirc
+echo "password: $PYPI_PASSWORD" >> $HOME/.pypirc
```

Comment:
IMO it would be cleaner not to echo so much and to use a single redirect instead of a double.

```
echo "[pipi]
repository: https://pypi.python.org/pypi
username: laslabs
password: derp" > $HOME/.pypirc 
```